lnwallet/channel.go

```diff
@@ -3970,7 +4001,7 @@ func (lc *LightningChannel) ProcessChanSyncMsg(
 // HTLCs will be set to the next commitment height.
 func (lc *LightningChannel) computeView(view *htlcView, remoteChain bool,
 	updateState bool) (lnwire.MilliSatoshi, lnwire.MilliSatoshi, int64,
-	*htlcView, error) {
+	btcutil.Amount, *htlcView, error) {
```

Comment:
Should update `godoc` with information re new return value. Also may just make the return value into a struct to avoid stuff like: 
```go
return 0, 0, 0, 0, nil, err
```

and also make things a bit more readable

Reply:
Done.